main/inc/lib/tracking.lib.php

```diff
@@ -1898,104 +1943,195 @@ public static function getLpStats(
+                                                $my_lesson_status = learnpathitem::humanize_status('incomplete');
+                                            }
+                                            $timeRow = '<td class="lp_time" colspan="2">'.$time_attemp.'</td>';
+                                            if ($hideTime) {
+                                                $timeRow = '';
+                                            }
 
-                                        $output .= '<tr class="'.$oddclass.'" >
-                                        <td></td>
-                                        <td>'.$extend_attempt_link.'</td>
-                                        <td colspan="3">'.get_lang('Attempt').' '.$n.'</td>
-                                        <td colspan="2">'.$my_lesson_status.'</td>
-                                        <td colspan="2">'.$view_score.'</td>
-                                        '.$timeRow;
-
-                                        if ($action == 'classic') {
-                                            if ($origin != 'tracking') {
-                                                if (!$is_allowed_to_edit && $result_disabled_ext_all) {
-                                                    $output .= '<td>
-                                                            <img src="'.Display::returnIconPath('quiz_na.gif').'" alt="'.get_lang('ShowAttempt').'" title="'.get_lang('ShowAttempt').'">
-                                                            </td>';
+                                            $output .= '<tr class="'.$oddclass.'" >
+                                    <td></td>
+                                    <td>'.$extend_attempt_link.'</td>
+                                    <td colspan="3">'.get_lang('Attempt').' '.$n.'</td>
+                                    <td colspan="2">'.$my_lesson_status.'</td>
+                                    <td colspan="2">'.$view_score.'</td>
+                                    '.$timeRow;
+
+                                            if ($action == 'classic') {
+                                                if ($origin != 'tracking') {
+                                                    if (!$is_allowed_to_edit && $result_disabled_ext_all) {
+                                                        $output .= '<td>
+                                                        <img src="'.Display::returnIconPath('quiz_na.gif').'" alt="'.get_lang('ShowAttempt').'" title="'.get_lang('ShowAttempt').'">
+                                                        </td>';
+                                                    } else {
+                                                        $output .= '<td>
+                                                        <a href="../exercise/exercise_show.php?origin='.$origin.'&id='.$my_exe_id.'&cidReq='.$courseCode.'" target="_parent">
```

Comment:
No session id ?

Reply:
I only reformatted that part of the code according to PHP Cs Fixer. It is exactly the same as it was before.